docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -781,6 +781,118 @@ To read and write a value of the Bluetooth descriptor:
+
+4. To register the callback called when a client reads the value of the characteristic from the local GATT server, create the callback function and pass it as an argument to the `setReadValueRequestCallback()` method called on the characteristic object.
+   ```
+   var characteristicReadRequestCallback = function(clientAddress, offset) {
+      console.log(clientAddress + " requested to read characteristic's value with offset: " + offset);
+      return new tizen.GATTRequestReply(0, "0x1234");
+   };
+   server.services[0].characteristics[0].setReadValueRequestCallback(characteristicReadRequestCallback);
+   ```
+
+   > **Note**
+   > A callback set with `setReadValueRequestCallback()` function overwrites any previously set `ReadValueRequestCallback` on this characteristic.
+
+5. To register the callback called when a client writes a value of the characteristic of the local GATT server, create the callback function and pass it as an argument to the `setWriteValueRequestCallback()` method called on the characteristic object.
+   ```
+   var characteristicWriteRequestCallback = function(clientAddress, value, offset, replyRequired) {
+      console.log(clientAddress + " requested to write characteristic's value: " + value +
+                  " with offset: " + offset);
+      return replyRequired ? new tizen.GATTRequestReply(0, "0x1234") : null;
+   };
+
+   server.services[0].characteristics[0].setWriteValueRequestCallback(characteristicWriteRequestCallback);
+   ```
+
+   > **Note**
+   > A callback set with `setWriteValueRequestCallback()` function overwrites any previously set `WriteValueRequestCallback` on this characteristic.
+
+6. To register the callback called when a client reads the value of the desriptor from the local GATT server, create the callback function and pass it as an argument to the `setReadValueRequestCallback()` method called on the desriptor object.
+   ```
+   var desriptorReadRequestCallback = function(clientAddress, offset) {
+      console.log(clientAddress + " requested to read desriptor's value with offset: " + offset);
+      return new tizen.GATTRequestReply(0, "0x1234");
+   };
+   server.services[0].characteristics[0].descriptors[0].setReadValueRequestCallback(characteristicReadRequestCallback);
+   ```
+
+   > **Note**
+   > A callback set with `setReadValueRequestCallback()` function overwrites any previously set `ReadValueRequestCallback` ot this descriptor.
+
+7. To register the callback called when a client writes a value of the desriptor of the local GATT server, create the callback function and pass it as an argument to the `setWriteValueRequestCallback()` method called on the desriptor object.
+   ```
+   var desriptorWriteRequestCallback = function(clientAddress, value, offset, replyRequired) {
+      console.log(clientAddress + " requested to write desriptor's value: " + value +
+                  " with offset: " + offset);
+      return replyRequired ? new tizen.GATTRequestReply(0, "0x1234") : null;
```

Comment:
Adding the second argument in GATTRequestReply only makes sense for read value callbacks. Please, remove it.

Reply:
done